right_front_wheel_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: left_front_wheel_controller_joint
    joint_speed: 4
    motor:
        id: 1

left_front_wheel_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: left_front_wheel_controller_joint
    joint_speed: 4
    motor:
        id: 2

right_rear_wheel_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: left_front_wheel_controller_joint
    joint_speed: 4
    motor:
        id: 3

left_rear_wheel_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: left_front_wheel_controller_joint
    joint_speed: 4
    motor:
        id: 4

